src/main/webapp/script.js

```diff
@@ -46,40 +48,27 @@ function cleanseUrl(url) {
  * Fetches data and adds to html
  */
 function getChart() {
-    const data = new google.visualization.DataTable();
-    data.addColumn('string', 'Range');
-    data.addColumn('number', 'Count');
-    // TODO: change hardcoded data to sentiment scores (once that part is finished)
-    data.addRows([
-        ['-1.0', null],
-        [null, 1],
-        ['-0.8', null],
-        [null, 1],
-        ['-0.6', null],
-        [null, 4],
-        ['-0.4', null],
-        [null, 5],
-        ['-0.2', null],
-        [null, 10],
-        ['0.0', null],
-        [null, 10],
-        ['0.2', null],
-        [null, 10],
-        ['0.4', null],
-        [null, 10],
-        ['0.6', null],
-        [null, 15],
-        ['0.8', null],
-        [null, 15],
-        ['1.0', null]
-    ]);
+  $('form').submit(function() {
+    const CommentCount = new google.visualization.DataTable();
+    CommentCount.addColumn('string', 'Range');
+    CommentCount.addColumn('number', 'Count');
```

Comment:
Sorry I should have been more explicit in my previous comment.

Can you change the name of the `Count` column to `CommentCount`?

Also maybe a better name for `Range` would be `SentimentRange`?

As for the name of the `DataTable` maybe something like `CommentSentimentTable` makes sense?

Reply:
Ah that makes more sense! :)